testing/matrix-sdk-integration-testing/src/tests/redaction.rs

```diff
@@ -1,4 +1,4 @@
-use anyhow::Result;
+use anyhow::{Ok, Result};
```

Comment:
This is the only change in this file, was it needed somehow?

Reply:
[`0390afd`](https://github.com/matrix-org/matrix-rust-sdk/pull/2118/commits/0390afd26d5d74018ef7c1d0a6eaa545386349ee)